training/mozilla_voice_stt_training/train.py

```diff
@@ -645,6 +645,9 @@ def __call__(self, progress, data, **kwargs):
                         log_info('Encountered a plateau, reducing learning rate to {}'.format(
                             current_learning_rate))
 
+                        # Reload checkpoint that we use the best_dev weights again
+                        load_or_init_graph_for_training(session, allow_drop_layers=False)
```

Comment:
Please keep this function unchanged and add a new explicitly `load_best_checkpoint` function, we don't want this call to load last silently for example.